src/services/mongoose/wallet-invoices.ts

```diff
@@ -144,5 +148,7 @@ const walletInvoiceFromRaw = (result): WalletInvoice => ({
   walletId: result.walletId as WalletId,
   selfGenerated: result.selfGenerated,
   pubkey: result.pubkey as Pubkey,
-  paid: result.paid,
+  paid: result.paid as boolean,
+  fiat: result.fiat as FiatAmount,
```

Comment:
This is supposed to be the fiat denomination correct? What about the display amount - are we going to persist that as well?

EG for Bitcoin Jungle we may have USD invoices that should still be displayed in colon.